tests/test_utilities/test_csvclean.py

```diff
@@ -18,6 +18,12 @@
 class TestCSVClean(CSVKitTestCase, EmptyFileTests):
     Utility = CSVClean
 
+    def tearDown(self):
```

Comment:
this change is irrelevant, but running this test file leaves behind a `stdin_out.csv` file. just cleaning it up!